src/router.debug.ts

```diff
@@ -1,17 +1,91 @@
+/* eslint-disable no-await-in-loop */
 /* eslint-disable no-console */
 import express, { Request, Response } from 'express';
 import { randomInt } from 'crypto';
-import { CustomerPartModel, ICustomerPart } from './customerPart/model';
-import { JobModel, IJob } from './job/model';
+import { CustomerPartModel } from './customerPart/model';
+import { JobModel } from './job/model';
 import { LotModel } from './lot/model';
 import { RouteStepModel } from './routeStep/model';
 import { RouteTemplateModel } from './routeTemplate/model';
+import { RouterModel } from './router/model';
+import { ICustomerPart, IJob, IRouter } from './global.interfaces';
 
 const Customer = require('./customer/model');
 
 const router = express.Router();
 router.post('/reset', resetData);
 
+router.post('/routers/reset', resetRouters);
```

Comment:
Can we have the `debug/reset` endpoint drop and reset routers, steps, etc as well, rather than having separate endpoints.